tests/unit/src/lib/validator.test.js

```diff
@@ -72,41 +78,61 @@ const bad_length_ninos = [
   "A",
   "",
 ];
+const invalid_character_ninos = ["abc def", "a b c d e f", "ab1cd 2"];
 
 describe("should fail all the bad ninos", () => {
   test.each(bad_prefixes_ninos)(
     "given bad prefix nino of %p, returns null string",
     (ninoArg) => {
-      const result = isValidNino(ninoArg);
-      expect(result).toBeNull();
+      testNinoValidation(ninoArg);
     }
   );
   test.each(bad_unused_prefixes_ninos)(
     "given unused prefix nino of %p, returns null string",
     (ninoArg) => {
-      const result = isValidNino(ninoArg);
-      expect(result).toBeNull();
+      testNinoValidation(ninoArg);
     }
   );
   test.each(bad_suffixes_ninos)(
     "given bad suffix nino of %p, returns null string",
     (ninoArg) => {
-      const result = isValidNino(ninoArg);
-      expect(result).toBeNull();
+      testNinoValidation(ninoArg);
     }
   );
   test.each(bad_length_ninos)(
     "given bad length nino of %p, returns null string",
     (ninoArg) => {
-      const result = isValidNino(ninoArg);
-      expect(result).toBeNull();
+      testNinoValidation(ninoArg);
+    }
+  );
+});
+
+describe("invoke the invalidCharacters function", () => {
+  test.each(invalid_character_ninos)(
+    "should return formattedValue when isValidNino does not return the same value for %p",
```

Comment:
Please can the emphasis not be on formattedValue, is there a better way to express this